lib/PlaidClient.js

```diff
@@ -79,6 +79,14 @@ Client.prototype._authenticatedRequest =
 Client.prototype.createPublicToken =
   requestWithAccessToken('/item/public_token/create', false);
 
+// createItemAddToken(Function)
+Client.prototype.createItemAddToken =
+  function(cb) {
+    return this._authenticatedRequest({
+      path: 'item/add_token/create',
```

Comment:
I think this should have a backslash before "item" - `/item/add_token/create`

Reply:
🤦‍♂ thanks!